sklearn/utils/_array_api.py

```diff
@@ -10,6 +10,10 @@
 from .fixes import parse_version
 
 
+def XP_FLOAT_DTYPES(xp):
```

Comment:
TODO
- [ ] docstring
- [ ] test
- [ ] better name?

Reply:
We already have `supported_float_dtypes`. I don't think we need a new function.